File: VB/MySolution.Module/BusinessObjects/BusinessModel.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace MySolution.Module.BusinessObjects
    <DefaultClassOptions> _
    Public Class Contact
        Inherits Person

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
    End Class
End Namespace

File: CS/MySolution.Module/BusinessObjects/BusinessModel.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace MySolution.Module.BusinessObjects {
    [DefaultClassOptions]
    public class Contact : Person {
        public Contact(Session session) :
            base(session) {
        }
    }
}